src/test/java/GuavaProgramming/FunctionalUtilitiesTest.java

```diff
@@ -0,0 +1,46 @@
+package GuavaProgramming;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by saurav on 9/7/16.
+ */
+public class FunctionalUtilitiesTest {
+    FunctionalUtilities funcUtils;
+    @Before
+    public void setUp() throws Exception {
+        funcUtils = new FunctionalUtilities();
+
+    }
+
+    @After
+    public void tearDown() throws Exception {
+
+    }
+
+    @Test
+    public void returnKeyValueTest() throws Exception {
+        Map<String, List> dataMap = Maps.newHashMap();
```

Comment:
May be better to use generics here.
Map<String, List<String>> dataMap = Maps.newHashMap();